crates/egui_extras/src/table.rs

```diff
@@ -288,17 +301,22 @@ impl<'a> Table<'a> {
             striped,
             clip,
             stick_to_bottom,
+            scroll_offset_y,
             cell_layout,
         } = self;
 
         let avail_rect = ui.available_rect_before_wrap();
 
         let mut new_widths = widths.clone();
 
-        let scroll_area = egui::ScrollArea::new([false, scroll])
+        let mut scroll_area = egui::ScrollArea::new([false, scroll])
             .auto_shrink([true; 2])
             .stick_to_bottom(stick_to_bottom);
 
+        if let Some(scroll_offset_y) = scroll_offset_y {
+            scroll_area = scroll_area.vertical_scroll_offset(scroll_offset_y);
```

Comment:
Just an idea: We could use `scroll_offset` using a `Vec2` and set the `x` component to `0` to be more future proof (if horizontal scroll will be available some day).

Reply:
I wouldn't. None means don't change the scroll position, whereas 0 means scroll to 0.